components/dev_to/dev_to.app.mjs

```diff
@@ -0,0 +1,40 @@
+import axios from "@pipedream/platform";
```

Comment:
Hummm, I don't know why this is working but normally I do the following import 
`import { axios } from "@pipedream/platform";`